src/wallet/wallet.cpp

```diff
@@ -303,6 +307,16 @@ bool CWallet::AddCryptedSaplingSpendingKey(const libzcash::SaplingFullViewingKey
         return true;
     {
         // TODO: Sapling - Write to disk
```

Comment:
Is this comment stale?

Reply:
Yes, it was used a place holder for code implementation. I will clean this up for final pass.